src/Abp.AspNetCore/AspNetCore/Configuration/AbpAspNetCoreConfiguration.cs

```diff
@@ -14,6 +15,10 @@ public class AbpAspNetCoreConfiguration : IAbpAspNetCoreConfiguration
 
         public IClientCacheAttribute DefaultClientCacheAttribute { get; set; }
```

Comment:
There is already some kind of implementation about it. 
https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5418/files#diff-0b9c8145133f735459eb77d08991f75dR42

But it was closed by @hikalkan about 3 years ago https://github.com/aspnetboilerplate/aspnetboilerplate/commit/a396051e74b7da646be8133ca0b40b0c2b0b0f90

Reply:
@demirmusa  i think when `ClientCacheAttribute` was implemented in [Abp 2.2](https://github.com/aspnetboilerplate/aspnetboilerplate/releases/tag/v2.2.0) via https://github.com/aspnetboilerplate/aspnetboilerplate/issues/2351, `ReponseCacheAttribute` was not available from `aspnetcore` since it was only released in aspnetcore 2.1.7. See https://github.com/dotnet/aspnetcore/commit/262bb9a7323b664d4b860931c04d1aea8142facd 

Abp only upgraded `aspnetcore` from 2.1.3 to 2.2 in [Abp 4.1](https://github.com/aspnetboilerplate/aspnetboilerplate/releases/tag/v4.1.0) via https://github.com/aspnetboilerplate/aspnetboilerplate/pull/4104

Reply:
Allright 👍 We can remove it in another pr since it is already comment outed.